src/v/pandaproxy/handlers.cc

```diff
@@ -221,4 +221,33 @@ subscribe_consumer(server::request_t rq, server::reply_t rp) {
       });
 }
 
+ss::future<server::reply_t>
+consumer_fetch(server::request_t rq, server::reply_t rp) {
+    auto fmt = parse_serialization_format(rq.req->get_header("Accept"));
+    if (fmt == ppj::serialization_format::unsupported) {
+        rp.rep = unprocessable_entity("Unsupported serialization format");
+        return ss::make_ready_future<server::reply_t>(std::move(rp));
+    }
+
+    std::chrono::milliseconds timeout{
+      boost::lexical_cast<std::chrono::milliseconds::rep>(
+        rq.req->get_query_param("timeout"))};
+    int32_t max_bytes{
+      boost::lexical_cast<int32_t>(rq.req->get_query_param("max_bytes"))};
```

Comment:
since this isn't a coroutine, should we catch lexical_cast throw's?

Reply:
All the handlers do it this way - there's an error handler installed: https://github.com/vectorizedio/redpanda/blob/dev/src/v/pandaproxy/server.cc#L144